encoder/libav_encoder.cpp

```diff
@@ -411,13 +434,15 @@ void LibAvEncoder::initOutput()
 
 void LibAvEncoder::deinitOutput()
 {
+
```

Comment:
Remove the blank line? (It's not that the blank line matters, it's just that diffs are easier to look at the smaller they are!!)